presto-main/src/main/java/io/prestosql/execution/QueryStateMachine.java

```diff
@@ -1127,12 +1127,12 @@ private static QueryStats pruneQueryStats(QueryStats queryStats)
         @GuardedBy("this")
         private boolean noMoreExchangeLocations;
 
-        public QueryOutputManager(Executor executor)
```

Comment:
Actually, the class should be private since it's an implementation detail of QueryStateMachine, and its methods should be public.